partner/WHMCS/modules/servers/jetpack/jetpack.php

```diff
@@ -71,18 +71,32 @@ function jetpack_ConfigOptions()
  */
 function jetpack_CreateAccount(array $params)
 {
-
-    validate_required_fields($params);
-    $access_token = get_access_token($params);
-    $response = provision_jetpack_plan($access_token, $params);
-
-    if (isset($response->next_url)) {
-        save_next_url($response->next_url, $params);
+    try {
+        validate_required_fields($params);
+
+        $access_token = get_access_token($params);
+        $provisioning_url = "https://public-api.wordpress.com/rest/v1.3/jpphp/provision";
+        $request_data = array (
+            'plan' => strtolower($params['customfields']['Plan']),
+            'siteurl' => $params['customfields']['Site URL'],
+            'local_user' => $params['customfields']['Local User'],
+            'force_register' => true,
+        );
+
+        $response = make_api_request($provisioning_url, $access_token, $request_data);
+
+        if (isset($response->next_url)) {
+            save_next_url($response->next_url, $params);
+        }
+
+        if ($response->success == true) {
+            return 'success';
+        }
+    } catch(Exception $e) {
```

Comment:
Why is `Exception` the best choice here?

Reply:
@nb based on earlier feedback about logModuleCall and discovery I've updated other areas to give feedback in the Activity Logs to Jetpack partner hosts on problems. Exceptions are also currently being caught and logModuleCall being used in the jetpack_createAccount and jetpack_terminateAccount functions only. I think catching an Exception here is the best choice to account for anything thats outside the realm of proper functioning of the module as well as the instances where exceptions are thrown were things may not go as intended specifically with the make_api_request function which does the provisioning/termination work. It's likely this should be outside the norm but since the create and terminate functions are core it seemed reasonable/necessary 🙂.